Sources/XcodeGen/main.swift

```diff
@@ -5,51 +5,52 @@ import XcodeGenKit
 import xcproj
 import ProjectSpec
 import JSONUtilities
-import Rainbow
 
 let version = "1.4.0"
 
-func generate(spec: String, project: String) {
+func generate(spec: String, project: String, isQuiet: Bool) {
+    let logger = Logger(isQuiet: isQuiet)
 
     let specPath = Path(spec).normalize()
     let projectPath = Path(project).normalize()
 
     if !specPath.exists {
-        print("No project spec found at \(specPath.absolute())".red)
-        exit(1)
+        logger.fatal("No project spec found at \(specPath.absolute())")
+		exit(1)
     }
 
     let spec: ProjectSpec
     do {
         spec = try ProjectSpec(path: specPath)
-        print("📋  Loaded spec:\n  \(spec.debugDescription.replacingOccurrences(of: "\n", with: "\n  "))")
+        logger.info("📋  Loaded spec:\n  \(spec.debugDescription.replacingOccurrences(of: "\n", with: "\n  "))")
     } catch let error as JSONUtilities.DecodingError {
-        print("Parsing spec failed: \(error.description)".red)
-        exit(1)
+        logger.fatal("Parsing spec failed: \(error.description)")
+		exit(1)
```

Comment:
spacing :trollface:

Reply:
Every time I regenerate the Xcode project it changes it back to tabs 🙄